autogpt/utils.py

```diff
@@ -11,10 +11,60 @@
 except:
     pass
 
+from autogpt.config import Config
 
-def clean_input(prompt: str = ""):
+cfg = Config()
+
+
+def clean_chat_message_to_user(report: str):
+    for plugin in enumerate(cfg.plugins):
+        if not plugin.can_handle_report():
+            continue
+        plugin.send_message(report)
+
+
+def clean_input(prompt: str = "", talk=False):
     try:
-        return input(prompt)
+        if cfg.chat_messages_enabled:
+            for plugin in enumerate(cfg.plugins):
+                if not plugin.can_handle_user_input():
```

Comment:
This potentially errors without the template package being updated, let's check that this attribute exists on the class before calling it.

Reply:
the other reports function needs this as well if it has been defined here.